pyrightconfig.json

```diff
@@ -22,4 +22,7 @@
     "reportUndefinedVariable": "warning",
     "reportOptionalOperand": "warning",
     "reportMissingImports": "warning",
+    "reportPrivateImportUsage": "warning",
```

Comment:
Why not fixing the single instance of this error (by properly importing the offending method)?

Reply:
I didn't see the immediate fix, and it did not seem worth it.

Reply:
I don't remember the names, but sage is importing x from y, and y is importing x from z (without reexporting it). So just import x from z.

Reply:
We don't even have such a thing as a convention of what reexports should look like in Sage. So warning instead of error is quite appropriate.

Reply:
Here `y` is an external library. So sage was accessing a non-public api. I think this is a good catch and should be fixed properly.

Reply:
I don't plan to work on this import. If you want to fix stuff, just open a PR.